templates/javascript.js.twig

```diff
@@ -5,6 +5,7 @@ App.createAddressSelector(
     '{{ address.city.vars.id }}',
     '{{ address.cityName.vars.id }}',
     '{{ address.cityName.vars.required }}',
+    '{{ address.region is defined ? address.region.vars.id : null }}',
```

Comment:
with this, you can pass `'null'` as a string value to the construct

Reply:
{{ address.region is defined ? address.region.vars.id|e('js') : null }},

{{ address.region is defined ? "'" address.region.vars.id|e('js') : null }},